advocacy_docs/edb-postgres-ai/ai-accelerator/retrievers/index.mdx

```diff
@@ -9,6 +9,6 @@ navigation:
 - example
 ---
 
-Retrievers can be used for semantic search or for RAG (retrieval augmented generation).
+Retrievers can be used for semantic search or for retrieval augmented generation (RAG).
 
-A configured retriever can automatically compute embeddings (vectors) for our input data. Once this is done, the retriever offers a number of interfaces to return data based on a query.
+A configured retriever can compute embeddings (vectors) for your input data. Once this is done, the retriever offers a number of interfaces to return data based on a query.
```

Comment:
Wasn't sure about the use of first person here but after reading everyhing a couple of times, I think just swapping it to second person is the intended meaning.